### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by zafields**
In order for the documentation to work like we say it does, these updates must be in place.

### CODE REVIEW

File: samples/nodejs_simple_sample/src/gateway_sample_lin.json [hunk 1]
```diff
@@ -25,23 +34,23 @@
             "loader": {
                 "name": "node",
                 "entrypoint": {
-                    "main.path": "<<path to iothub_writer.js>>"
+                    "main.path": "../../../../samples/nodejs_simple_sample/nodejs_modules/iothub_writer.js"
                 }
             },
             "args": {
-                "connection_string": "<<IoT Hub Device Connection String>>"
+                "connection_string": "<IoTHub device connection string>"
```
Comment (damonbarry) [inline samples/nodejs_simple_sample/src/gateway_sample_lin.json]: It should be IoT Hub...that's the official name (you'll see them squished together in variable names)

File: samples/nodejs_simple_sample/src/gateway_sample_lin.json [hunk 2]
```diff
@@ -1,11 +1,20 @@
 {
+    "loaders": [
```
Comment (damonbarry) [inline samples/nodejs_simple_sample/src/gateway_sample_lin.json]: Is the loaders section necessary? If this section is optional then I'd rather leave it out of the sample. We want to demonstrate a *minimal* working example...

File: samples/nodejs_simple_sample/src/gateway_sample_win.json [hunk 3]
```diff
@@ -34,11 +34,11 @@
             "loader": {
                 "name": "node",
                 "entrypoint": {
-                    "main.path": "../../../samples/nodejs_simple_sample/nodejs_modules/iothub_writer.js"
+                    "main.path": "../../../../samples/nodejs_simple_sample/nodejs_modules/iothub_writer.js"
                 }
             },
             "args": {
-                "connection_string": "<<IoT Hub Device Connection String>>"
+                "connection_string": "<IoTHub device connection string>"
```
Comment (damonbarry) [inline samples/nodejs_simple_sample/src/gateway_sample_win.json]: IoT<space>Hub

File: tools/build_nodejs.cmd [hunk 4]
```diff
@@ -48,7 +48,31 @@ copy node\build\Release\lib\*.lib dist\lib
...
+echo.    set NODE_INCLUDE=%build-root%\dist\inc\
+
+echo.    set NODE_LIB=%build-root%\dist\lib\
+
+echo.
+
+
+
+set NODE_INCLUDE=%build-root%\dist\inc\
+set NODE_LIB=%build-root%\dist\lib\
```
Comment (damonbarry) [inline tools/build_nodejs.cmd]: Are you going to add `endlocal`?

Comment (zafields) [inline tools/build_nodejs.cmd]: Yes. As soon as I test it.

File: samples/nodejs_simple_sample/src/gateway_sample_win.json [hunk 5]
```diff
@@ -14,7 +14,7 @@
             "loader": {
                 "name": "node",
                 "entrypoint": {
-                    "main.path": "../../../samples/nodejs_simple_sample/nodejs_modules/printer.js"
```
Comment (zafields) [inline samples/nodejs_simple_sample/src/gateway_sample_win.json]: This is the difference between running it from the command line and running it from VS.

File: samples/nodejs_simple_sample/src/gateway_sample_lin.json (hunk 2 above)
Comment (zafields) [inline samples/nodejs_simple_sample/src/gateway_sample_lin.json]: This was given to me as is by Raj. There was conversation around this at scrum that sounded like a firm "maybe".

**Pull request closed without merging**